packages/babel-compiler/package.js

```diff
@@ -1,13 +1,13 @@
 Package.describe({
   name: "babel-compiler",
   summary: "Parser/transpiler for ECMAScript 2015+ syntax",
-  version: '7.11.2',
+  version: '7.11.3',
 });
 
 Npm.depends({
-  '@meteorjs/babel': '7.20.1',
-  'json5': '2.1.1',
-  'semver': '7.3.8'
+  '@9morello/babel-dev': '8.0.1',
+  'json5': '2.2.3',
+  'semver': '7.6.3'
```

Comment:
I thought these changes were part of the dev_bundle tool and needed verification with a devbundle, but that's not the case. We can cherry-pick them to verify the bumps and release them directly.